crates/parcel_plugin_rpc/src/nodejs/worker_init.rs

```diff
@@ -0,0 +1,60 @@
+use std::sync::mpsc::channel;
+use std::sync::mpsc::Receiver;
+use std::sync::mpsc::Sender;
+use std::thread;
+
+use once_cell::sync::Lazy;
+
+use crate::RpcConnectionMessage;
+
+enum WorkerInitMessage {
+  Subscribe(Sender<Sender<RpcConnectionMessage>>),
+  Register(Sender<RpcConnectionMessage>),
+}
+
+// This allows for workers to notify RPC Connections that they are ready
+static WORKER_INIT: Lazy<Sender<WorkerInitMessage>> = Lazy::new(|| {
```

Comment:
shouldn't be static, but I'm not sure what this is achieving

can we just call the thread-safe function on the Rpc worker? why do we need channels at all if thread-safe function already wraps a queue?

Reply:
> shouldn't be static
What should it be? `const`?

I could play around with using tsfns more directly though this abstraction behind channels is nicer to interact with